packages/api/src/core/MutatorDescriptor.ts

```diff
@@ -1,6 +1,9 @@
+import { ParserPlugin } from '@babel/parser';
+
 interface MutatorDescriptor {
   name: string;
   excludedMutations: string[];
+  babelPlugins: ParserPlugin[];
```

Comment:
Could we rename it to `plugins` (of type `string[]`)? That way we can also reuse it whenever typescript would allow plugins.